src/main.rs

```diff
@@ -327,8 +327,9 @@ fn compile_executable(file_name: &str, host: &host::Host) -> Result<(), io::Erro
     process::exit(1);
   }
 
-  let target = format!("./{}", file_name.strip_suffix(".hvmc").unwrap_or(file_name));
-  fs::copy("./.hvm/target/release/hvmc", target)?;
+  let target = format!("{}", file_name.strip_suffix(".hvmc").unwrap_or(file_name));
```

Comment:
is this `format!()` necessary?

Reply:
It isn't, I'm just used to using it unnecessarily.